### DISCUSSION TITLE
Name is not saved into the database

### DISCUSSION THREAD

**Discussion by plompd** [spatie/laravel-medialibrary#2246] [General]
Hi, I'm using the MediaLibraryPro package inside my Laravel project (v8). If I use the <x-media-library> the Name of the file that is being uploaded is not saved into the database?

This is a screenshot after I save my model:

![image](https://user-images.githubusercontent.com/1482375/103747140-7e0efc00-5002-11eb-98f4-1a0cb576438e.png)

So the temporary upload seems okay, but the final image not. Is this a known issue?

**Comment by freekmurze** [comment]
No, it's not a know issue, could you try and see if it works in our demo app?
https://github.com/spatie/laravel-medialibrary-pro-app

If it does, compare our app to your app.
